source/Engine.cpp

```diff
@@ -725,10 +725,6 @@ void Engine::Step(bool isActive)
 	{
 		target = flagship->GetTargetShip();
 		targetAsteroid = flagship->GetTargetAsteroid();
-		// Record that the player knows this type of asteroid is available here.
-		if(targetAsteroid)
-			for(const auto &it : targetAsteroid->Payload())
-				player.Harvest(it.first);
```

Comment:
Why remove this? Targeting no longer maps the asteroid as being in this system?

Reply:
With the auto catalog I saw no reason to keep it.  Just by being in the vicinity it is catalogged.